### ISSUE TITLE
Config font validating improvements

### ISSUE THREAD

**Issue by The-Compiler** [The-Compiler/qutebrowser#103]
font validation and parsing rejects some valid values or accepts multiple families.

We should use `pyPEG2` instead to parse it properly

**Comment by The-Compiler** [comment]
Maybe this would work?

```
>>> from PyQt5.QtWidgets import QApplication
>>> app = QApplication([])
>>> from PyQt5.QtGui import QFont
>>> QFont("asfojas").exactMatch()
False
>>> QFont("Terminus").exactMatch()
True
```

See [font matching algorithm](http://doc.qt.io/qt-5/qfont.html#fontmatching)

**Comment by The-Compiler** [comment]
Merged into #115.
